docs/layers/scenegraph-layer.md

```diff
@@ -71,6 +71,10 @@ Can be a URL of an object. You need to provide the `fetch` function to load the
 Can also be a luma.gl [ScenegraphNode](http://uber.github.io/luma.gl/#/documentation/api-reference/), or a `Promise` that resolves to one.
 The layer calls _delete()_ on _scenegraph_ when a new one is provided or the layer is finalized.
 
+##### `scenegraphLoader` (Object, optional)
+
+If you provide a URL for `scenegraph` you can optionally provide a `scenegraphLoader` to load the URL.
```

Comment:
Describe what a loader is, where I can get one etc. by linking to loaders.gl documentation.

Reply:
Done